scripts/linode-bootstrap.sh

```diff
@@ -82,7 +82,7 @@ label_disks() {
 ################################################################################
 # Download the NixOS ISO and write it to the installer disk.
 write_installer_to_disk() {
-  curl -k "$NIXOS_URL" \
+  curl -L -k "$NIXOS_URL" \
```

Comment:
I'm to blame as well.  I tested the URL on my end and should have said something when I needed to add the `-L` to my invocation of `curl`.  Oh well.